README.md

```diff
@@ -744,6 +836,11 @@ end
 
 ### Type: cisco_vtp
 
+| Minimum Requirements | N9k | N30xx | N31xx | N56xx | N6k | N7k |
```

Comment:
cisco_vtp is not supported on all of these platforms right?  We should not list minimum requirements if not supported.  Chris has a similar issue for puppet so sync with him on the use of an `unsupported` key word.